.changeset/fifty-geese-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": minor
```

Comment:
Shouldn't be a patch?

Reply:
If you think this is a patch, then I'd change the "Types of changes" in the PR description to "Bug fix".
Then, please fill the template with missing information.